docs/plugins.rst

```diff
@@ -166,6 +166,9 @@ plugin under consideration.  The objects themselves are relatively simple:
       enabled: false
       path: path/to/quux
 
+    - name: docs
+      enabled: true
+
```

Comment:
Ah I was confusing the main conf file with the plugin conf file. All seems to work as expected now. Is there a good reason to not merge these two with a `plugins` list as a field in the main conf?